policyengine_us/parameters/gov/states/va/tax/income/filing_requirement.yaml

```diff
@@ -0,0 +1,25 @@
+description: Virginia requires the filer to file an income tax returen with state adjusted gross income at or above this limit.
+metadata:
+  label: Virginia filing requirement
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: Code of Virginia § 58.1-321. Exemptions and exclusions. (A)
```

Comment:
Mention that the legal code also adds a deduction to this but it's not in the tax forms